kafka-producer/src/main/resources/config/base.conf

```diff
@@ -1,13 +1,32 @@
 {
   "haystack": {
-    "directory": "extractors/"
+    "kafka": {
+      "brokers": "localhost",
+      "port": 9092,
+      "fromtopic": "proto-spans",
+      "totopic": "",
+      "threadcount": 1,
+      "sessiontimeout": 15000,
+      "maxwakeups": 10,
+      "wakeuptimeoutms": 3000,
+      "polltimeoutms": 250,
+      "commitms": 3000
+    },
+    "externalKafkaList": [{
```

Comment:
Comment what each config does (except the obvious ones like kafka ones). Try to be as verbose as possible.